### ISSUE TITLE
Show Organisation API usage when running self hosted postgres analytics

### ISSUE THREAD

**Issue by dabeeeenster** [Flagsmith/flagsmith#2016]
Not sure where this would best live - if its best in Flagsmith on Flagsmith or elsewhere?

<img width="879" alt="image" src="https://user-images.githubusercontent.com/173290/224751702-3ce0e001-1abb-4c73-9a17-578df7d968dd.png">

**Comment by matthewelwell** [comment]
@dabeeeenster can you flesh this issue out? I'm not entirely sure what's missing / needs adding.

**Comment by dabeeeenster** [comment]
I don't think this tab shows up at all if you are running with a basic self hosted setup, e.g. docker-compose.
